src/transformStateWithClones.js

```diff
@@ -6,8 +6,40 @@
  *
  * @return {Object[]}
  */
+const ADD_PROPERTIES = 'addProperties';
+const REMOVE_PROPERTIES = 'removeProperties';
+const CLEAR = 'clear';
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  let tempState = { ...state };
```

Comment:
[NAMING] - Instead of naming the object copy as 'tempState', a more descriptive name such as 'stateCopy' would better convey the purpose of this variable.